### PR TITLE
Add a guide for mysqldump

### PR DESCRIPTION
**Pull request by morgo**
This helps with a common use case.

Signed-off-by: Morgan Tocker <tocker@gmail.com>

### CODE REVIEW

File: content/en/docs/user-guides/export-data.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+	---
+title: Export data from Vitess
```
Comment (bnlandry) [inline content/en/docs/user-guides/export-data.md]: I think we should use 'Exporting...' for consistency with other user guide titles.

File: content/en/docs/user-guides/export-data.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+weight: 8
+---
+
+Since [VTGate](../../concepts/vtgate/) supports the MySQL protocol, in many cases it is possible to use existing client utilities when connecting to Vitess.  This includes using logical dump tools such as `mysqldump` or `mydumper`.
```
Comment (bnlandry) [inline content/en/docs/user-guides/export-data.md]: Nit: extra space before "This includes..."

File: content/en/docs/user-guides/export-data.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+This guide provides instructions on the required options when using these tools against a VTGate server for the purposes of exporting data from Vitess. It is recommended to follow the [Backup and Restore](../backup-and-restore/) guide for regular backups, since this method is performed directly on the tablet servers and is more efficient for larger databases. 
+
+### mysqldump
+
+The default invocation of `mysqldump` attempts to execute statements which are [not supported by Vitess](../../reference/mysql-compatibility/), such as attempting to lock tables and dump GTID coordinates. The following options are required when using mysqldump from MySQL 5.7 to export data from the `commerce` keyspace:
+
```
Comment (bnlandry) [inline content/en/docs/user-guides/export-data.md]: It might help clarity to first list the options in general, then give the example.

**Review by bnlandry — APPROVED** [review]

**Comment by morgo** [comment]
I am happy with the contents of this, but would like to perform more testing before merging.

**Comment by deepthi** [comment]
Merged as-is after checking with @morgo. We can improve it based on feedback.

**Pull request merged**